plugin/server/config/index.js

```diff
@@ -22,13 +22,33 @@ module.exports = {
     translateRelations,
   }) {
     if (provider === 'dummy') {
-      console.warn(
-        'provider is set to dummy by default. This only copies all values'
-      )
+      // console.warn(
```

Comment:
Why is this commented out?

Reply:
Commented it out since it polluted the tests, have now added the node env to the if